### DISCUSSION TITLE
Applying Suggested Fixes

### DISCUSSION THREAD

**Discussion by StevenACoffman** [golangci/golangci-lint#2346] [Q&A] [answered]
Hi. At Khan made https://github.com/golangci/golangci-lint/pull/2300 to apply suggested fixes (especially for custom linters) and have been using it for several months prior with no issues. Is there an alternative in the works somewhere? Any ideas for further improvement? I'm not really clear what the hold up on merging it as is was for. We are happy to make whatever suggested changes and are eager to get this (or something equivalent) merged. @ldez can you at least give us an estimate as to **when** you'll be able to explain your reasoning for blocking it?

**Comment by ldez** [comment, marked as answer]
You pinged on that topic on Slack, on the PR, and now in a discussion, are you sure if it's the right way to contribute?
The previous week, I spent a lot of time on the preparation of the release. Please remember that I am an open-source volunteer, I work on this project in my spare time.

I will make a quick answer on your PR.

**Comment by StevenACoffman** [comment]
Thanks for your comment on the PR. Sorry for making all the noise, but contributing working PRs seemed like the correct way to contribute. It seems to me like waiting for an undesigned, alternative, possible future solution that may be more comprehensive would not need to hold up merging a currently working improvement.

Your past volunteer involvement is certainly greater than mine or my colleagues. I just didn't want to be kept waiting in suspense indefinitely. I'll close the current PR, and we'll continue to carry it in our fork, so there's no need to further rush your design or add stress to your volunteer commitment.
